bin/reth/src/args/miner_args.rs

```diff
@@ -0,0 +1,78 @@
+use clap::Args;
+use reth_primitives::Address;
+
+/// Parameters for configuring the miner
+#[derive(Debug, Args, PartialEq, Default)]
+pub struct MinerArgs {
+    /// Enable mining
+    #[arg(long = "mine", help_heading = "Miner")]
+    pub mine: bool,
```

Comment:
We can remove this as well since mining is no longer a thing?

Reply:
I believe so, this should be a setting of CL